### PR TITLE
Update api versions of kubernetes manifests

### PR DESCRIPTION
**Pull request by johscheuer**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug

/kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature

**What this PR does / why we need it**:
Update api versions of kubernetes manifests and also add the labels to all resources.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: build/cloud/08-service.yaml.example [hunk 1]
```diff
@@ -7,6 +7,7 @@ metadata:
     k8s-app: kubeedge
     kubeedge: cloudcore
 spec:
+  type: NodePort
```
Comment (chendave) [inline build/cloud/08-service.yaml.example]: why did this change here?

Comment (fisherxu) [reply build/cloud/08-service.yaml.example]: I think we can use `hostNetwork` in deployment

Comment (johscheuer) [reply build/cloud/08-service.yaml.example]: This was only a mistake, I did this change for my local setup.

File: build/cloud/01-namespace.yaml [hunk 2]
```diff
@@ -2,3 +2,5 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: kubeedge
+  labels:
+    kubeedge: cloudcore
```
Comment (fisherxu) [inline build/cloud/01-namespace.yaml]: Why do we need this label? :) And the `kubeedge` ns will be used for all the components for kubeedge cloudside, not only for cloudcore here.

Comment (johscheuer) [reply build/cloud/01-namespace.yaml]: That's right I wanted to add the `k8s-app: kubeedge` label. Thanks for catching this one !

**Comment by fisherxu** [comment]
/lgtm
Thanks!

**Pull request merged**